sysid-projects/analysis-test/src/main/include/Elevator.h

```diff
@@ -77,6 +77,5 @@ class Elevator : public SysIdGeneralMechanism {
                                             1000,
                                             2_in,
                                             -Constants::Elevator::kHeight,
-                                            Constants::Elevator::kHeight,
-                                            true};
+                                            Constants::Elevator::kHeight};
```

Comment:
This shouldn't have been changed (not in this PR at least).

Reply:
It fixed the build on my end, since elevator sim has added the option to simulate gravity since release 2022.4.1. I can take it out, though.

Reply:
Ah, didn't realize that. Then it should be fine.